Makefile

```diff
@@ -12,15 +14,29 @@ compile:
 doc:
 	$(REBAR) doc $(REBAR_FLAGS)
 
-test: compile
-	$(REBAR) eunit $(REBAR_FLAGS)
+test: compile eunit ct
+
+## Run code checks and unit+integration tests
+test_and_verify: verify_format lint dialyzer test
```

Comment:
I've created a new target `make test_and_verify` and the idea is to get it to pass all checks in order to have this PR ready to be merged!